unified_planning/transformers/ma_negative_conditions_remover.py

```diff
@@ -0,0 +1,210 @@
+# Copyright 2021 AIPlan4EU project
```

Comment:
Instead of an entirely new calss we shall try to refactor/reuse existing code to avoid a huge duplication